src/data.toml

```
@@ -1,17 +1,18 @@
 no_mod_settings = "Try visiting [this link](<https://raw.githubusercontent.com/pyoncord/detta-builds/main/bunny.js>) and see if it loads bunch of texts. If it doesn't, try using a VPN and/or disable any sort of ad blocker and try again."
 custom_fonts = "Fonts are experimental and some packs may break the application. You can get font packs listed here: https://discord.com/channels/1196075698301968455/1225584192012947506/1241453802184380418"
 ios_notifications = "For notifications to work on a Jailed device you need to sign the ipa with a certificate that has the notification entitlement enabled.\n\nPlease keep in mind that this channel is not sideloading support.\nPlease keep questions pertaining to this topic elsewhere."
-thirty_two_bit_android_os = "32-bit versions of Android OS are not supported. Please use a 64-bit operating system."
+thirty_two_bit_android_os = "32-bit versions of Android OS are not supported. Please use a 64-bit operating system.\n### Use a System Info App:\nDownload a system information app from the Google Play Store, like "CPU-Z" or "AIDA64." \nThese apps provide detailed information about your device's hardware, including the CPU architecture. \n### Terminal Emulator (Rooted Devices):If your device is rooted, you can use a terminal emulator."
 broken_addons = "You can find a curated list of broken add-ons in https://discord.com/channels/1196075698301968455/1210652540757090315/1229376326528663596"
 altstore_pal = "AltStore PAL is an alternate app marketplace for notarized applications. You cannot use it to sideload.\n\nPlease keep in mind that this channel is not sideloading support.\nPlease keep questions pertaining to this topic elsewhere."
 no_passkeys_when_pre_patched = "Passkeys are not supported when using the pre-patched application. Please use a different method to log in to your account or use the LSPosed module / iOS tweak."
 ios_app_icons = "When sideloading with a paid certificate the application will be signed using a wildcard App ID.\nThis effectively breaks being able to change app icons.\nYou need to sign the application with a local dev cert instead if you want this functionality.\n\nPlease keep in mind that this channel is not sideloading support.\nPlease keep questions pertaining to this topic elsewhere."
-addon_404 = "You are not supposed to visit the url directly since it does not serve anything that can be rendered. Use it to install the add-on."
+addon_404 = "You are NOT supposed to visit the url directly since it does not serve anything that can be rendered. Copy the link and use it to install the add-on."
+not_support_channel = "### You are using the wrong channel!\nThis is not the right channel to ask for any support or requests, please consider going in https://discord.com/channels/1196075698301968455/1210652540757090315 for those."
```

this should probably be formulated a bit more grammatically eloquent
```suggestion
not_support_channel = "This is not the support channel, please move to https://discord.com/channels/1196075698301968455/1210652540757090315."
```